SPECS/initramfs/initramfs.spec

```diff
@@ -88,15 +88,17 @@ echo "initramfs (re)generation" %* >&2
 cat > /dev/null \
 if [ -f %{_localstatedir}/lib/rpm-state/initramfs/regenerate ]; then \
     echo "(re)generate initramfs for all kernels," %* >&2 \
-    mkinitrd -q \
+    mkinitrd --preload megaraid_sas --preload megaraid -q \
```

Comment:
I have concnerns about forcing these drivers on all images. dracut has ways to be more selective using conf files.

Off the top of my head I can think of:
Option 1: Using a post-install script, add the drivers to the dracut conf file, then trigger a rebuild of the  initramfs
Option 2: Add a package that includes a dedicated conf file in dracut.conf.d/

Following the path from --preload, it looks like this maps to just adding a driver to dracut normally. Dracut also offers a force_drivers option  but I haven't used it personally.

Reply:
I had to add some drivers as part of the verity module: https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/verity-read-only-root/verity.conf

Reply:
I like option 2, but I am not inclined to add a top-level package for this.. Would another option be to add a subpackage to dracut? It would be similar to what dracut-fips currently does.

Reply:
That seems like a valid option.